CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Added `compas_rhino.objects`.
 * Added `compas_rhino.layers`.
 * Added `compas_rhino.install_with_pip`.
+* Added `__eq__` to `compas.data.Data`.
```

Comment:
If this gets added, I would suggest the changelog entry to be more explicit about what is happening, because this was rather extensive consequences, which are not evident by reading this sentence.